generators/contract/templates/java/src/main/java/org/example/MyAsset.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * <%= spdxAndLicense // SPDX-License-Identifier: Apache-2.0 %>
+ */
+
+package org.example;
+
+import org.hyperledger.fabric.contract.annotation.Contract;
+import org.hyperledger.fabric.contract.annotation.DataType;
+import org.hyperledger.fabric.contract.annotation.Property;
+import org.json.JSONObject;
+
+@DataType()
+public class <%= assetPascalCase %> {
+
+    @Property()
+    private String value;
+
+    public <%= assetPascalCase %>(){
```

Comment:
Do we really need the empty constructor?

Reply:
Not strictly according to the Language spec...Following JavaBean guidelines here - and as this is a type it makes it a lot easier to recreate the object.